plugins/api/include/api_plugin_number_data.h

```diff
@@ -8,3 +8,4 @@ API_PLUGIN_NUMBER(ATOMIC_APPLY_ACL_OPERATIONS_APN,              20005)
 API_PLUGIN_NUMBER(DATA_OBJECT_FINALIZE_APN,                     20006)
 API_PLUGIN_NUMBER(TOUCH_APN,                                    20007)
 API_PLUGIN_NUMBER(REGISTER_PHYSICAL_PATH_APN,                   20008)
+API_PLUGIN_NUMBER(GET_DELAY_RULE_INFO_APN,                      20009)
```

Comment:
Heads up that `GET_GRID_CONFIGURATION_VALUE_APN` has value 20009 on main/4-3-stable.

An interesting situation that I'm not sure we have encountered as of yet...

Reply:
Most definitely.

Reply:
looks like 20013 is in your future...

https://github.com/irods/irods/blob/7a8ea89af09a3ab222f8009825dda0b207328735/plugins/api/include/irods/plugins/api/api_plugin_number_data.h#L15-L16

Reply:
Done.

Reply:
I think we need a comment explaining the gap in API plugin numbers. I just don't know where I want to put that. Hmm ...

Reply:
just add it right there and say 4.3.0 already has 9-12.

Reply:
I considered that, but hesitated because that file is changed based on the definition of the `API_PLUGIN_NUMBER` macro.

I'll thinking on it.

Reply:
Added comment explaining the gap.